packages/client/src/admin/Images/ImageForm.tsx

```diff
@@ -250,7 +250,7 @@ function _ImageForm({
       }
     }
 
-    if (data?.urlForGpu) {
+    if (data?.type == 'both' && data?.urlForGpu !== data?.url) {
```

Comment:
Is make sense to compare with `urlForGpu` and `url`? I remember if `urlForGpu` not given value, it will be `data.url` because the backend side auto assign the value.

Reply:
This is original design. 

If the image is universal and gpu is not specified, both cpu and gpu url will have the same value.